osquery/logger/plugins/kafka_producer.cpp

```diff
@@ -26,17 +23,44 @@
 namespace osquery {
 
 FLAG(string,
-     logger_kafka_brokers,
+     kafka_brokers,
      "localhost",
      "Bootstrap broker(s) as a comma-separated list of host or host:port "
      "(default port 9092)");
+FLAG_ALIAS(std::string, logger_kafka_brokers, kafka_brokers);
 
-FLAG(string, logger_kafka_topic, "", "Kafka topic to publish logs under");
+FLAG(string, kafka_topic, "osquery", "Kafka topic to publish logs under");
+FLAG_ALIAS(std::string, logger_kafka_topic, kafka_topic);
 
 FLAG(string,
-     logger_kafka_acks,
+     kafka_acks,
      "all",
      "The number of acknowledgments the leader has to receive (0, 1, 'all')");
+FLAG_ALIAS(std::string, logger_kafka_acks, kafka_acks);
+
+FLAG(string,
+     kafka_security_protocol,
+     "plaintext",
+     "Protocol used to communicate with brokers ('plaintext', 'ssl')");
+
+FLAG(string, kafka_ssl_cipher_suites, "", "List of cipher suites");
```

Comment:
Unless there are specific use cases for not doing so, I'd like to use a hard-coded set of suites: https://github.com/facebook/osquery/blob/master/osquery/remote/transports/tls.cpp#L25

Reply:
What about using these values as default and letting user override them if necessary?

Reply:
That sounds fair if you know you'll need this flexibility. If you do provide it, can we use 1 flag for all osquery TLS uses? Right now that constant string is used for AWS/TLS plugins.